SampleSitePdks/pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[project]
+name = "hdl21-sitepdks"
```

Comment:
Keep the name `sitepdks` plz

Reply:
ok, updated.  Please approve the CI run.